### PR TITLE
Controls

### PR DESCRIPTION
**Pull request by pkoch**
Nothing too fancy. Notice you can double jump for now. :P

### CODE REVIEW

**Review by goncalossilva — APPROVED** [review]

File: moonshot.p8 [hunk 1]
```diff
@@ -42,19 +41,46 @@ function init_player()
...
     -plr.max_dy,
     plr.dy+g,
     plr.max_dy
    )
-   
-   
+
+   -- Apply frct to dx
+   if plr.dx != 0 then
+     if plr.flp then -- going left
```
Comment (goncalossilva) [inline moonshot.p8:62]: Indent should be 1 space, the built-in editor is 32 characters wide. :sweat_smile:

File: moonshot.p8 [hunk 2]
```diff
@@ -42,19 +41,46 @@ function init_player()
...
    --apply gravity
    plr.dy=mid(
     -plr.max_dy,
     plr.dy+g,
     plr.max_dy
    )
-   
-   
+
+   -- Apply frct to dx
```
Comment (goncalossilva) [inline moonshot.p8:60]: Please avoid capitals, the built-in editor displays them smaller than regular letters.

File: moonshot.p8 [hunk 3]
```diff
@@ -42,19 +41,46 @@ function init_player()
...
+   if plr.dx != 0 then
```
Comment (goncalossilva) [inline moonshot.p8:61]: Please avoid spaces in-between tokens... 32 characters wide, etc.

**Comment by goncalossilva** [comment]
Works great! Double jump really is multi-jump. :grin:

Left a few comments, but all related with style, mostly from using the built-in editor. @jgradim and I discussed some of these today.

**Comment by pkoch** [comment]
Yeah, I was using vim, so my normal.coding habits kicked in. Will try to be more careful next time. 👌

**Pull request merged**